library/src/main/java/com/applandeo/materialcalendarview/utils/DateUtils.kt

```diff
@@ -0,0 +1,121 @@
+@file:JvmName("DateUtils")
+
+package com.applandeo.materialcalendarview.utils
+
+import android.content.Context
+import com.applandeo.materialcalendarview.R
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+/**
+ * Created by Mateusz Kornakiewicz on 23.05.2017.
+ */
+
+/**
+ * @return An instance of the Calendar object with hour set to 00:00:00:00
+ */
+val getMidnightCalendar: Calendar = Calendar.getInstance().apply {
```

Comment:
midnightCalendar and check if in java code you can use it as getMidnightCalendar()

Reply:
Yes it can be done